ci-operator/config/openshift/release/openshift-release-master__ocp-4.8.yaml

```diff
@@ -211,6 +211,18 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-libvirt-cert-rotation
+- artifact_dir: /tmp/artifacts
+  as: e2e-aws-single-node
+  cron: 0 0 * * 1
```

Comment:
is it the same time when all assisted periodic jobs are running? Maybe to split so there will be less tress on the packet servers